packages/driver/src/cy/keyboard.ts

```diff
@@ -681,7 +681,9 @@ export class Keyboard {
               const keysToType = keyDetailsArr.slice(currentKeyIndex, currentKeyIndex + _skipCheckUntilIndex)
 
               _.each(keysToType, (key) => {
-                key.simulatedDefaultOnly = true
```

Comment:
this is what fixed the bug, setting this prop prevents keyboard events from being sent